splinterd/src/daemon.rs

```diff
@@ -920,14 +921,20 @@ fn create_node_registry(
     registry_config: &RegistryConfig,
 ) -> Result<Box<dyn RwNodeRegistry>, RestApiServerError> {
     match registry_config {
-        RegistryConfig::File { registry_file } => Ok(Box::new(
-            node_registry::yaml::YamlNodeRegistry::new(&registry_file).map_err(|err| {
-                RestApiServerError::StartUpError(format!(
-                    "Failed to initialize YamlNodeRegistry: {}",
-                    err
-                ))
-            })?,
-        )),
+        RegistryConfig::File { registry_file } => {
+            debug!(
+                "Creating node registry with registry file: {:?}",
+                fs::canonicalize(&registry_file)?
```

Comment:
Do these paths get check if they exist before they are logged? If not we should not use ? and instead catch the error and add which path failed in the error message

Reply:
Yes, it is checked here  https://github.com/Cargill/splinter/blob/master/splinterd/src/main.rs#L282
and set to None if the file doesn't exist before it gets to this point.

Reply:
Same with all of the other config files, all are checked with `is_file` before being logged.